zk.js/src/test-utils/setUpMerkleTree.ts

```diff
@@ -0,0 +1,127 @@
+import * as anchor from "@coral-xyz/anchor";
+import { PublicKey } from "@solana/web3.js";
+import {
+  IDL_LIGHT_PSP2IN2OUT,
+  IDL_LIGHT_PSP10IN2OUT,
+  IDL_LIGHT_PSP4IN4OUT_APP_STORAGE,
+  IDL_LIGHT_PSP2IN2OUT_STORAGE,
+} from "../idls/index";
+
+import {
+  ADMIN_AUTH_KEYPAIR,
+  POOL_TYPE,
+  MINT,
+  merkleTreeProgramId,
+  airdropSol,
+  lightPsp2in2outId,
+  lightPsp10in2outId,
+  lightPsp4in4outAppStorageId,
+  lightPsp2in2outStorageId,
+  getSignerAuthorityPda,
+} from "../index";
+import { MerkleTreeConfig } from "../merkle-tree/merkle-tree-config";
+
+export async function setUpMerkleTree(
```

Comment:
setup considered one word 

```suggestion
export async function setupMerkleTree(
```